src/util/Enums.js

```diff
@@ -16,6 +16,7 @@ export default {
   WIDGET_LOGIN_CONTAINER_ID: 'okta-login-container',
   WIDGET_CONTAINER_ID: 'okta-sign-in',
   WIDGET_CONFIG_COLORS_ID: 'okta-sign-in-config-colors',
+  WIDGET_FOOTER_CLASS: 'siw-main-footer',
```

Comment:
Just wondering why we'd opt for this rather than keeping it here. The other widget related selectors are in this file, to me it feels inconsistent if we just move this one out to `Constants.js`

Reply:
Trying to separate from V1 as much as possible. probably only `WIDGET_CONTAINER_ID` are shared between both v1 and v2.